From: Kay Sievers <kay@vrfy.org>
Date: Sun, 13 Apr 2014 23:46:27 -0700
Subject: hwdb: update

(cherry picked from commit dbe633cc027e31a0d4fb488a3c68b7762ef0e7b3)
(cherry picked from commit 6ece7006e35a434b961cbed397e9f71bf3b5c47e)
---
 hwdb/20-OUI.hwdb              | 275 +++++++++++++++++++++-
 hwdb/20-pci-vendor-model.hwdb | 522 ++++++++++++++++++++++++++++++++++++++----
 2 files changed, 751 insertions(+), 46 deletions(-)

diff --git a/hwdb/20-OUI.hwdb b/hwdb/20-OUI.hwdb
index 209b89e..17794ed 100644
--- a/hwdb/20-OUI.hwdb
+++ b/hwdb/20-OUI.hwdb
@@ -6929,7 +6929,7 @@ OUI:0050C2912*
  ID_OUI_FROM_DATABASE=ASSET InterTech, Inc.
 
 OUI:0050C2913*
- ID_OUI_FROM_DATABASE=Selex Sensors & Airborne Systems
+ ID_OUI_FROM_DATABASE=Selex Ex Ltd
 
 OUI:0050C2914*
  ID_OUI_FROM_DATABASE=IO-Connect
@@ -15944,7 +15944,7 @@ OUI:000300*
  ID_OUI_FROM_DATABASE=Barracuda Networks, Inc.
 
 OUI:000301*
- ID_OUI_FROM_DATABASE=Avantas Networks Corporation
+ ID_OUI_FROM_DATABASE=EXFO
 
 OUI:000302*
  ID_OUI_FROM_DATABASE=Charles Industries, Ltd.
@@ -24575,7 +24575,7 @@ OUI:000E65*
  ID_OUI_FROM_DATABASE=TransCore
 
 OUI:000E66*
- ID_OUI_FROM_DATABASE=Hitachi Advanced Digital, Inc.
+ ID_OUI_FROM_DATABASE=Hitachi Industry & Control Solutions, Ltd.
 
 OUI:000E67*
  ID_OUI_FROM_DATABASE=Eltis Microelectronics Ltd.
@@ -24947,7 +24947,7 @@ OUI:000EE1*
  ID_OUI_FROM_DATABASE=ExtremeSpeed Inc.
 
 OUI:000EE2*
- ID_OUI_FROM_DATABASE=Custom Engineering S.p.A.
+ ID_OUI_FROM_DATABASE=Custom Engineering
 
 OUI:000EE3*
  ID_OUI_FROM_DATABASE=Chiyu Technology Co.,Ltd
@@ -51760,6 +51760,9 @@ OUI:045A95*
 OUI:045C06*
  ID_OUI_FROM_DATABASE=Zmodo Technology Corporation
 
+OUI:045C8E*
+ ID_OUI_FROM_DATABASE=gosund GROUP CO.,LTD
+
 OUI:045D56*
  ID_OUI_FROM_DATABASE=camtron industrial inc.
 
@@ -51865,6 +51868,9 @@ OUI:04C5A4*
 OUI:04C880*
  ID_OUI_FROM_DATABASE=Samtec Inc
 
+OUI:04C991*
+ ID_OUI_FROM_DATABASE=Phistek INC.
+
 OUI:04CB1D*
  ID_OUI_FROM_DATABASE=Traka plc
 
@@ -52633,6 +52639,9 @@ OUI:08CA45*
 OUI:08CC68*
  ID_OUI_FROM_DATABASE=Cisco
 
+OUI:08CD9B*
+ ID_OUI_FROM_DATABASE=samtec automotive electronics & software GmbH
+
 OUI:08D09F*
  ID_OUI_FROM_DATABASE=CISCO SYSTEMS, INC.
 
@@ -52684,6 +52693,9 @@ OUI:08F2F4*
 OUI:08F6F8*
  ID_OUI_FROM_DATABASE=GET Engineering
 
+OUI:08F728*
+ ID_OUI_FROM_DATABASE=GLOBO Multimedia Sp. z o.o. Sp.k.
+
 OUI:08FAE0*
  ID_OUI_FROM_DATABASE=Fohhn Audio AG
 
@@ -52750,6 +52762,9 @@ OUI:0C3021*
 OUI:0C37DC*
  ID_OUI_FROM_DATABASE=Huawei Technologies Co., Ltd
 
+OUI:0C383E*
+ ID_OUI_FROM_DATABASE=Fanvil Technology Co., Ltd.
+
 OUI:0C3956*
  ID_OUI_FROM_DATABASE=Observator instruments
 
@@ -53104,6 +53119,9 @@ OUI:102D96*
 OUI:102EAF*
  ID_OUI_FROM_DATABASE=Texas Instruments
 
+OUI:103047*
+ ID_OUI_FROM_DATABASE=Samsung Electronics Co.,Ltd
+
 OUI:103378*
  ID_OUI_FROM_DATABASE=FLECTRON Co., LTD
 
@@ -53215,6 +53233,9 @@ OUI:1088CE*
 OUI:108CCF*
  ID_OUI_FROM_DATABASE=CISCO SYSTEMS, INC.
 
+OUI:109266*
+ ID_OUI_FROM_DATABASE=Samsung Electronics Co.,Ltd
+
 OUI:1093E9*
  ID_OUI_FROM_DATABASE=Apple
 
@@ -53260,6 +53281,9 @@ OUI:10BF48*
 OUI:10C2BA*
  ID_OUI_FROM_DATABASE=UTT Co., Ltd.
 
+OUI:10C37B*
+ ID_OUI_FROM_DATABASE=ASUSTek COMPUTER INC.
+
 OUI:10C586*
  ID_OUI_FROM_DATABASE=BIO SOUND LAB CO., LTD.
 
@@ -53365,6 +53389,9 @@ OUI:141BBD*
 OUI:141BF0*
  ID_OUI_FROM_DATABASE=Intellimedia Systems Ltd
 
+OUI:141FBA*
+ ID_OUI_FROM_DATABASE=IEEE REGISTRATION AUTHORITY  - Please see MAM public listing for more information.
+
 OUI:1423D7*
  ID_OUI_FROM_DATABASE=EUTRONIX CO., LTD.
 
@@ -53518,6 +53545,9 @@ OUI:14B968*
 OUI:14C089*
  ID_OUI_FROM_DATABASE=DUNE HD LTD
 
+OUI:14C126*
+ ID_OUI_FROM_DATABASE=Nokia Corporation
+
 OUI:14C21D*
  ID_OUI_FROM_DATABASE=Sabtech Industries
 
@@ -53572,6 +53602,9 @@ OUI:14F28E*
 OUI:14F42A*
  ID_OUI_FROM_DATABASE=Samsung Electronics
 
+OUI:14F65A*
+ ID_OUI_FROM_DATABASE=Xiaomi inc.
+
 OUI:14FEAF*
  ID_OUI_FROM_DATABASE=SAGITTAR LIMITED
 
@@ -53659,6 +53692,9 @@ OUI:182B05*
 OUI:182C91*
  ID_OUI_FROM_DATABASE=Concept Development, Inc.
 
+OUI:183009*
+ ID_OUI_FROM_DATABASE=Woojin Industrial Systems Co., Ltd.
+
 OUI:1832A2*
  ID_OUI_FROM_DATABASE=LAON TECHNOLOGY CO., LTD.
 
@@ -53698,6 +53734,9 @@ OUI:184617*
 OUI:1848D8*
  ID_OUI_FROM_DATABASE=Fastback Networks
 
+OUI:184A6F*
+ ID_OUI_FROM_DATABASE=Alcatel-Lucent Shanghai Bell Co., Ltd
+
 OUI:184E94*
  ID_OUI_FROM_DATABASE=MESSOA TECHNOLOGIES INC.
 
@@ -53758,6 +53797,9 @@ OUI:1880F5*
 OUI:188331*
  ID_OUI_FROM_DATABASE=Samsung Electronics Co.,Ltd
 
+OUI:1883BF*
+ ID_OUI_FROM_DATABASE=Arcadyan Technology Corporation
+
 OUI:188410*
  ID_OUI_FROM_DATABASE=CoreTrust Inc.
 
@@ -54322,6 +54364,9 @@ OUI:206A8A*
 OUI:206AFF*
  ID_OUI_FROM_DATABASE=Atlas Elektronik UK Limited
 
+OUI:206E9C*
+ ID_OUI_FROM_DATABASE=Samsung Electronics Co.,Ltd
+
 OUI:206FEC*
  ID_OUI_FROM_DATABASE=Braemac CA LLC
 
@@ -54838,6 +54883,9 @@ OUI:284C53*
 OUI:284D92*
  ID_OUI_FROM_DATABASE=Luminator
 
+OUI:284ED7*
+ ID_OUI_FROM_DATABASE=OutSmart Power Systems, Inc.
+
 OUI:284FCE*
  ID_OUI_FROM_DATABASE=Liaoning Wontel Science and Technology Development Co.,Ltd.
 
@@ -54976,6 +55024,9 @@ OUI:28C68E*
 OUI:28C718*
  ID_OUI_FROM_DATABASE=Altierre
 
+OUI:28C7CE*
+ ID_OUI_FROM_DATABASE=Cisco
+
 OUI:28C825*
  ID_OUI_FROM_DATABASE=DellKing Industrial Co., Ltd
 
@@ -55069,6 +55120,9 @@ OUI:28FBD3*
 OUI:28FC51*
  ID_OUI_FROM_DATABASE=The Electric Controller and Manufacturing Co., LLC
 
+OUI:28FCF6*
+ ID_OUI_FROM_DATABASE=Shenzhen Xin KingBrand enterprises Co.,Ltd
+
 OUI:2C002C*
  ID_OUI_FROM_DATABASE=UNOWHY
 
@@ -55291,6 +55345,9 @@ OUI:2CB693*
 OUI:2CB69D*
  ID_OUI_FROM_DATABASE=RED Digital Cinema
 
+OUI:2CBE08*
+ ID_OUI_FROM_DATABASE=Apple
+
 OUI:2CBE97*
  ID_OUI_FROM_DATABASE=Ingenieurbuero Bickele und Buehler GmbH
 
@@ -55354,6 +55411,9 @@ OUI:30055C*
 OUI:300B9C*
  ID_OUI_FROM_DATABASE=Delta Mobile Systems, Inc.
 
+OUI:300D2A*
+ ID_OUI_FROM_DATABASE=Zhejiang Wellcom Technology Co.,Ltd.
+
 OUI:300ED5*
  ID_OUI_FROM_DATABASE=Hon Hai Precision Ind.Co.Ltd
 
@@ -55450,6 +55510,9 @@ OUI:3055ED*
 OUI:3057AC*
  ID_OUI_FROM_DATABASE=IRLAB LTD.
 
+OUI:30595B*
+ ID_OUI_FROM_DATABASE=streamnow AG
+
 OUI:3059B7*
  ID_OUI_FROM_DATABASE=Microsoft
 
@@ -55543,6 +55606,9 @@ OUI:30B3A2*
 OUI:30B5C2*
  ID_OUI_FROM_DATABASE=TP-LINK TECHNOLOGIES CO.,LTD.
 
+OUI:30B5F1*
+ ID_OUI_FROM_DATABASE=Aitexin Technology Co., Ltd
+
 OUI:30C750*
  ID_OUI_FROM_DATABASE=MIC Technology Group
 
@@ -55606,6 +55672,9 @@ OUI:30F9ED*
 OUI:30FD11*
  ID_OUI_FROM_DATABASE=MACROTECH (USA) INC.
 
+OUI:340286*
+ ID_OUI_FROM_DATABASE=Intel Corporate
+
 OUI:3407FB*
  ID_OUI_FROM_DATABASE=Ericsson AB
 
@@ -55654,6 +55723,9 @@ OUI:342F6E*
 OUI:343111*
  ID_OUI_FROM_DATABASE=Samsung Electronics Co.,Ltd
 
+OUI:3431C4*
+ ID_OUI_FROM_DATABASE=AVM GmbH
+
 OUI:3440B5*
  ID_OUI_FROM_DATABASE=IBM
 
@@ -55840,6 +55912,9 @@ OUI:34C059*
 OUI:34C3AC*
  ID_OUI_FROM_DATABASE=Samsung Electronics
 
+OUI:34C5D0*
+ ID_OUI_FROM_DATABASE=Hagleitner Hygiene International GmbH
+
 OUI:34C69A*
  ID_OUI_FROM_DATABASE=Enecsys Ltd
 
@@ -55963,6 +56038,9 @@ OUI:3826CD*
 OUI:3828EA*
  ID_OUI_FROM_DATABASE=Fujian Netcom Technology Co., LTD
 
+OUI:382DD1*
+ ID_OUI_FROM_DATABASE=Samsung Electronics Co.,Ltd
+
 OUI:3831AC*
  ID_OUI_FROM_DATABASE=WEG
 
@@ -56371,6 +56449,9 @@ OUI:3C99F7*
 OUI:3C9F81*
  ID_OUI_FROM_DATABASE=Shenzhen CATIC Bit Communications Technology Co.,Ltd
 
+OUI:3CA10D*
+ ID_OUI_FROM_DATABASE=Samsung Electronics Co.,Ltd
+
 OUI:3CA315*
  ID_OUI_FROM_DATABASE=Bless Information & Communications Co., Ltd
 
@@ -56380,6 +56461,9 @@ OUI:3CA72B*
 OUI:3CA9F4*
  ID_OUI_FROM_DATABASE=Intel Corporate
 
+OUI:3CAA3F*
+ ID_OUI_FROM_DATABASE=iKey, Ltd.
+
 OUI:3CAB8E*
  ID_OUI_FROM_DATABASE=Apple
 
@@ -56413,6 +56497,9 @@ OUI:3CC99E*
 OUI:3CCA87*
  ID_OUI_FROM_DATABASE=Iders Incorporated
 
+OUI:3CCD5A*
+ ID_OUI_FROM_DATABASE=Technische Alternative GmbH
+
 OUI:3CCD93*
  ID_OUI_FROM_DATABASE=LG ELECTRONICS INC
 
@@ -57109,6 +57196,9 @@ OUI:481A84*
 OUI:481BD2*
  ID_OUI_FROM_DATABASE=Intron Scientific co., ltd.
 
+OUI:4826E8*
+ ID_OUI_FROM_DATABASE=Tek-Air Systems, Inc.
+
 OUI:48282F*
  ID_OUI_FROM_DATABASE=ZTE Corporation
 
@@ -57260,7 +57350,7 @@ OUI:48D7FF*
  ID_OUI_FROM_DATABASE=BLANKOM Antennentechnik GmbH
 
 OUI:48D855*
- ID_OUI_FROM_DATABASE=IEEE REGISTRATION AUTHORITY
+ ID_OUI_FROM_DATABASE=Telvent
 
 OUI:48D8FE*
  ID_OUI_FROM_DATABASE=ClarIDy Solutions, Inc.
@@ -57748,6 +57838,9 @@ OUI:507E5D*
 OUI:508569*
  ID_OUI_FROM_DATABASE=Samsung Electronics Co.,LTD
 
+OUI:508789*
+ ID_OUI_FROM_DATABASE=Cisco
+
 OUI:5087B8*
  ID_OUI_FROM_DATABASE=Nuvyyo Inc
 
@@ -57880,6 +57973,9 @@ OUI:50FC30*
 OUI:50FC9F*
  ID_OUI_FROM_DATABASE=Samsung Electronics Co.,Ltd
 
+OUI:50FEF2*
+ ID_OUI_FROM_DATABASE=Sify Technologies Ltd
+
 OUI:5403F5*
  ID_OUI_FROM_DATABASE=EBN Technology Corp.
 
@@ -58072,6 +58168,9 @@ OUI:54AE27*
 OUI:54B620*
  ID_OUI_FROM_DATABASE=SUHDOL E&C Co.Ltd.
 
+OUI:54B753*
+ ID_OUI_FROM_DATABASE=Hunan Fenghui Yinjia Science And Technology Co.,Ltd
+
 OUI:54BEF7*
  ID_OUI_FROM_DATABASE=PEGATRON CORPORATION
 
@@ -58132,6 +58231,9 @@ OUI:54F5B6*
 OUI:54F666*
  ID_OUI_FROM_DATABASE=Berthold Technologies GmbH and Co.KG
 
+OUI:54FA3E*
+ ID_OUI_FROM_DATABASE=Samsung Electronics Co.,LTD
+
 OUI:54FB58*
  ID_OUI_FROM_DATABASE=WISEWARE, Lda
 
@@ -58462,6 +58564,9 @@ OUI:5C260A*
 OUI:5C2AEF*
  ID_OUI_FROM_DATABASE=Open Access Pty Ltd
 
+OUI:5C2E59*
+ ID_OUI_FROM_DATABASE=Samsung Electronics Co.,Ltd
+
 OUI:5C313E*
  ID_OUI_FROM_DATABASE=Texas Instruments
 
@@ -58519,6 +58624,9 @@ OUI:5C57C8*
 OUI:5C5948*
  ID_OUI_FROM_DATABASE=Apple
 
+OUI:5C5BC2*
+ ID_OUI_FROM_DATABASE=YIK Corporation
+
 OUI:5C5EAB*
  ID_OUI_FROM_DATABASE=Juniper Networks
 
@@ -58558,6 +58666,9 @@ OUI:5C8778*
 OUI:5C89D4*
  ID_OUI_FROM_DATABASE=Beijing Banner Electric Co.,Ltd
 
+OUI:5C8D4E*
+ ID_OUI_FROM_DATABASE=Apple
+
 OUI:5C95AE*
  ID_OUI_FROM_DATABASE=Apple
 
@@ -58819,6 +58930,9 @@ OUI:607688*
 OUI:60812B*
  ID_OUI_FROM_DATABASE=Custom Control Concepts
 
+OUI:6081F9*
+ ID_OUI_FROM_DATABASE=Helium Systems, Inc
+
 OUI:6083B2*
  ID_OUI_FROM_DATABASE=GkWare e.K.
 
@@ -58867,6 +58981,9 @@ OUI:60A10A*
 OUI:60A44C*
  ID_OUI_FROM_DATABASE=ASUSTek COMPUTER INC.
 
+OUI:60A8FE*
+ ID_OUI_FROM_DATABASE=Nokia Solutions and Networks
+
 OUI:60A9B0*
  ID_OUI_FROM_DATABASE=Merchandising Technologies, Inc
 
@@ -58918,6 +59035,9 @@ OUI:60C980*
 OUI:60CBFB*
  ID_OUI_FROM_DATABASE=AirScape Inc.
 
+OUI:60CDA9*
+ ID_OUI_FROM_DATABASE=Abloomy
+
 OUI:60CDC5*
  ID_OUI_FROM_DATABASE=Taiwan Carol Electronics., Ltd
 
@@ -59092,6 +59212,9 @@ OUI:644D70*
 OUI:644F74*
  ID_OUI_FROM_DATABASE=LENUS Co., Ltd.
 
+OUI:644FB0*
+ ID_OUI_FROM_DATABASE=Hyunjin.com
+
 OUI:64517E*
  ID_OUI_FROM_DATABASE=LONG BEN (DONGGUAN) ELECTRONIC TECHNOLOGY CO.,LTD.
 
@@ -59146,6 +59269,9 @@ OUI:64680C*
 OUI:6469BC*
  ID_OUI_FROM_DATABASE=Hytera Communications Co .,ltd
 
+OUI:646CB2*
+ ID_OUI_FROM_DATABASE=Samsung Electronics Co.,Ltd
+
 OUI:646E6C*
  ID_OUI_FROM_DATABASE=Radio Datacom LLC
 
@@ -59305,6 +59431,9 @@ OUI:64D814*
 OUI:64D912*
  ID_OUI_FROM_DATABASE=Solidica, Inc.
 
+OUI:64D954*
+ ID_OUI_FROM_DATABASE=TAICANG AND W ELECTRONICS CO LTD
+
 OUI:64D989*
  ID_OUI_FROM_DATABASE=CISCO SYSTEMS, INC.
 
@@ -59368,6 +59497,9 @@ OUI:64F987*
 OUI:64FC8C*
  ID_OUI_FROM_DATABASE=Zonar Systems
 
+OUI:680571*
+ ID_OUI_FROM_DATABASE=Samsung Electronics Co.,Ltd
+
 OUI:6805CA*
  ID_OUI_FROM_DATABASE=Intel Corporate
 
@@ -59701,6 +59833,9 @@ OUI:6C2E33*
 OUI:6C2E85*
  ID_OUI_FROM_DATABASE=SAGEMCOM
 
+OUI:6C2F2C*
+ ID_OUI_FROM_DATABASE=Samsung Electronics Co.,Ltd
+
 OUI:6C32DE*
  ID_OUI_FROM_DATABASE=Indieon Technologies Pvt. Ltd.
 
@@ -59860,6 +59995,9 @@ OUI:6CADF8*
 OUI:6CAE8B*
  ID_OUI_FROM_DATABASE=IBM Corporation
 
+OUI:6CB0CE*
+ ID_OUI_FROM_DATABASE=NETGEAR
+
 OUI:6CB311*
  ID_OUI_FROM_DATABASE=Shenzhen Lianrui Electronics Co.,Ltd
 
@@ -59875,6 +60013,9 @@ OUI:6CBEE9*
 OUI:6CC1D2*
  ID_OUI_FROM_DATABASE=ARRIS Group, Inc.
 
+OUI:6CC217*
+ ID_OUI_FROM_DATABASE=Hewlett Packard
+
 OUI:6CC26B*
  ID_OUI_FROM_DATABASE=Apple
 
@@ -59929,6 +60070,9 @@ OUI:6CF97C*
 OUI:6CFA58*
  ID_OUI_FROM_DATABASE=Avaya, Inc
 
+OUI:6CFA89*
+ ID_OUI_FROM_DATABASE=Cisco
+
 OUI:6CFAA7*
  ID_OUI_FROM_DATABASE=AMPAK Technology Inc.
 
@@ -60097,6 +60241,9 @@ OUI:7071B3*
 OUI:7071BC*
  ID_OUI_FROM_DATABASE=PEGATRON CORPORATION
 
+OUI:70720D*
+ ID_OUI_FROM_DATABASE=Lenovo Mobile Communication Technology Ltd.
+
 OUI:70723C*
  ID_OUI_FROM_DATABASE=Huawei Technologies Co., Ltd
 
@@ -60502,6 +60649,9 @@ OUI:74D850*
 OUI:74DA38*
  ID_OUI_FROM_DATABASE=Edimax Technology Co. Ltd.
 
+OUI:74DBD1*
+ ID_OUI_FROM_DATABASE=Ebay Inc
+
 OUI:74DE2B*
  ID_OUI_FROM_DATABASE=Liteon Technology Corporation
 
@@ -60733,6 +60883,9 @@ OUI:78923E*
 OUI:78929C*
  ID_OUI_FROM_DATABASE=Intel Corporate
 
+OUI:789684*
+ ID_OUI_FROM_DATABASE=ARRIS Group, Inc.
+
 OUI:7898FD*
  ID_OUI_FROM_DATABASE=Q9 Networks Inc.
 
@@ -60922,6 +61075,9 @@ OUI:78F7BE*
 OUI:78F7D0*
  ID_OUI_FROM_DATABASE=Silverbrook Research
 
+OUI:78FD94*
+ ID_OUI_FROM_DATABASE=Apple
+
 OUI:78FE3D*
  ID_OUI_FROM_DATABASE=Juniper Networks
 
@@ -61060,6 +61216,9 @@ OUI:7C69F6*
 OUI:7C6AB3*
  ID_OUI_FROM_DATABASE=IBC TECHNOLOGIES INC.
 
+OUI:7C6AC3*
+ ID_OUI_FROM_DATABASE=GatesAir, Inc
+
 OUI:7C6ADB*
  ID_OUI_FROM_DATABASE=SafeTone Technology Co.,Ltd
 
@@ -61087,6 +61246,9 @@ OUI:7C6F06*
 OUI:7C6FF8*
  ID_OUI_FROM_DATABASE=ShenZhen ACTO Digital Video Technology Co.,Ltd.
 
+OUI:7C70BC*
+ ID_OUI_FROM_DATABASE=IEEE REGISTRATION AUTHORITY  - Please see MAM public listing for more information.
+
 OUI:7C72E4*
  ID_OUI_FROM_DATABASE=Unikey Technologies
 
@@ -61174,6 +61336,9 @@ OUI:7CBFB1*
 OUI:7CC3A1*
  ID_OUI_FROM_DATABASE=Apple
 
+OUI:7CC4EF*
+ ID_OUI_FROM_DATABASE=Devialet
+
 OUI:7CC537*
  ID_OUI_FROM_DATABASE=Apple
 
@@ -61231,6 +61396,9 @@ OUI:7CE044*
 OUI:7CE1FF*
  ID_OUI_FROM_DATABASE=Computer Performance, Inc. DBA Digital Loggers, Inc.
 
+OUI:7CE524*
+ ID_OUI_FROM_DATABASE=Quirky, Inc.
+
 OUI:7CE56B*
  ID_OUI_FROM_DATABASE=ESEN Optoelectronics Technology Co.,Ltd.
 
@@ -61477,6 +61645,9 @@ OUI:80A1D7*
 OUI:80AAA4*
  ID_OUI_FROM_DATABASE=USAG
 
+OUI:80AD67*
+ ID_OUI_FROM_DATABASE=Kasda Digital Technology Co.,Ltd
+
 OUI:80B219*
  ID_OUI_FROM_DATABASE=ELEKTRON TECHNOLOGY UK LIMITED
 
@@ -61564,6 +61735,9 @@ OUI:80FFA8*
 OUI:8400D2*
  ID_OUI_FROM_DATABASE=Sony Ericsson Mobile Communications AB
 
+OUI:8401A7*
+ ID_OUI_FROM_DATABASE=Greyware Automation Products, Inc
+
 OUI:840B2D*
  ID_OUI_FROM_DATABASE=SAMSUNG ELECTRO-MECHANICS CO., LTD
 
@@ -61576,6 +61750,9 @@ OUI:841715*
 OUI:841766*
  ID_OUI_FROM_DATABASE=Weifang GoerTek Electronics Co., Ltd
 
+OUI:84183A*
+ ID_OUI_FROM_DATABASE=Ruckus Wireless
+
 OUI:841888*
  ID_OUI_FROM_DATABASE=Juniper Networks
 
@@ -61723,6 +61900,9 @@ OUI:848433*
 OUI:848506*
  ID_OUI_FROM_DATABASE=Apple
 
+OUI:84850A*
+ ID_OUI_FROM_DATABASE=Hella Sonnen- und Wetterschutztechnik GmbH
+
 OUI:848D84*
  ID_OUI_FROM_DATABASE=Rajant Corporation
 
@@ -61885,6 +62065,9 @@ OUI:8823FE*
 OUI:88252C*
  ID_OUI_FROM_DATABASE=Arcadyan Technology Corporation
 
+OUI:882950*
+ ID_OUI_FROM_DATABASE=Dalian Netmoon Tech Develop Co.,Ltd
+
 OUI:882E5A*
  ID_OUI_FROM_DATABASE=storONE
 
@@ -61948,6 +62131,9 @@ OUI:885C47*
 OUI:88615A*
  ID_OUI_FROM_DATABASE=Siano Mobile Silicon Ltd.
 
+OUI:8863DF*
+ ID_OUI_FROM_DATABASE=Apple
+
 OUI:88685C*
  ID_OUI_FROM_DATABASE=Shenzhen ChuangDao & Perpetual Eternal Technology Co.,Ltd
 
@@ -62098,6 +62284,9 @@ OUI:88E917*
 OUI:88ED1C*
  ID_OUI_FROM_DATABASE=Cudo Communication Co., Ltd.
 
+OUI:88F031*
+ ID_OUI_FROM_DATABASE=Cisco
+
 OUI:88F077*
  ID_OUI_FROM_DATABASE=CISCO SYSTEMS, INC.
 
@@ -62326,6 +62515,9 @@ OUI:8CB864*
 OUI:8CBEBE*
  ID_OUI_FROM_DATABASE=Xiaomi Technology Co.,Ltd
 
+OUI:8CBF9D*
+ ID_OUI_FROM_DATABASE=Shanghai Xinyou Information Technology Ltd. Co.
+
 OUI:8CC121*
  ID_OUI_FROM_DATABASE=Panasonic Corporation AVC Networks Company
 
@@ -62662,6 +62854,9 @@ OUI:90B97D*
 OUI:90C115*
  ID_OUI_FROM_DATABASE=Sony Ericsson Mobile Communications AB
 
+OUI:90C792*
+ ID_OUI_FROM_DATABASE=ARRIS Group, Inc.
+
 OUI:90CC24*
  ID_OUI_FROM_DATABASE=Synaptics, Inc
 
@@ -63259,6 +63454,9 @@ OUI:98EC65*
 OUI:98F0AB*
  ID_OUI_FROM_DATABASE=Apple
 
+OUI:98F170*
+ ID_OUI_FROM_DATABASE=Murata Manufacturing Co., Ltd.
+
 OUI:98F537*
  ID_OUI_FROM_DATABASE=zte corporation
 
@@ -63496,6 +63694,9 @@ OUI:9CA577*
 OUI:9CA9E4*
  ID_OUI_FROM_DATABASE=zte corporation
 
+OUI:9CAD97*
+ ID_OUI_FROM_DATABASE=Hon Hai Precision Ind. Co.,Ltd.
+
 OUI:9CADEF*
  ID_OUI_FROM_DATABASE=Obihai Technology, Inc.
 
@@ -64456,6 +64657,9 @@ OUI:A8E018*
 OUI:A8E3EE*
  ID_OUI_FROM_DATABASE=Sony Computer Entertainment Inc.
 
+OUI:A8E539*
+ ID_OUI_FROM_DATABASE=Moimstone Co.,Ltd
+
 OUI:A8EF26*
  ID_OUI_FROM_DATABASE=Tritonwave
 
@@ -64702,6 +64906,9 @@ OUI:ACA016*
 OUI:ACA22C*
  ID_OUI_FROM_DATABASE=Baycity Technologies Ltd
 
+OUI:ACA31E*
+ ID_OUI_FROM_DATABASE=Aruba Networks
+
 OUI:ACA430*
  ID_OUI_FROM_DATABASE=Peerless AV
 
@@ -64837,6 +65044,12 @@ OUI:B000B4*
 OUI:B00594*
  ID_OUI_FROM_DATABASE=Liteon Technology Corporation
 
+OUI:B009D3*
+ ID_OUI_FROM_DATABASE=Avizia
+
+OUI:B01041*
+ ID_OUI_FROM_DATABASE=Hon Hai Precision Ind. Co.,Ltd.
+
 OUI:B01203*
  ID_OUI_FROM_DATABASE=Dynamics Hong Kong Limited
 
@@ -64951,6 +65164,9 @@ OUI:B0808C*
 OUI:B081D8*
  ID_OUI_FROM_DATABASE=I-sys Corp
 
+OUI:B083FE*
+ ID_OUI_FROM_DATABASE=Dell Inc
+
 OUI:B0869E*
  ID_OUI_FROM_DATABASE=Chloride S.r.L
 
@@ -65122,6 +65338,9 @@ OUI:B407F9*
 OUI:B40832*
  ID_OUI_FROM_DATABASE=TC Communications
 
+OUI:B40B44*
+ ID_OUI_FROM_DATABASE=Smartisan Technology Co., Ltd.
+
 OUI:B40B7A*
  ID_OUI_FROM_DATABASE=Brusa Elektronik AG
 
@@ -65200,6 +65419,9 @@ OUI:B4417A*
 OUI:B4430D*
  ID_OUI_FROM_DATABASE=Broadlink Pty Ltd
 
+OUI:B4475E*
+ ID_OUI_FROM_DATABASE=Avaya, Inc
+
 OUI:B44CC2*
  ID_OUI_FROM_DATABASE=NR ELECTRIC CO., LTD
 
@@ -65608,6 +65830,9 @@ OUI:B8A8AF*
 OUI:B8AC6F*
  ID_OUI_FROM_DATABASE=Dell Inc
 
+OUI:B8AD3E*
+ ID_OUI_FROM_DATABASE=BLUECOM
+
 OUI:B8AE6E*
  ID_OUI_FROM_DATABASE=Nintendo Co., Ltd.
 
@@ -65842,6 +66067,9 @@ OUI:BC4B79*
 OUI:BC4E3C*
  ID_OUI_FROM_DATABASE=CORE STAFF CO., LTD.
 
+OUI:BC4E5D*
+ ID_OUI_FROM_DATABASE=ZhongMiao Technology Co., Ltd.
+
 OUI:BC51FE*
  ID_OUI_FROM_DATABASE=Swann Communications Pty Ltd
 
@@ -65974,6 +66202,9 @@ OUI:BCC6DB*
 OUI:BCC810*
  ID_OUI_FROM_DATABASE=Cisco SPVTG
 
+OUI:BCCAB5*
+ ID_OUI_FROM_DATABASE=ARRIS Group, Inc.
+
 OUI:BCCD45*
  ID_OUI_FROM_DATABASE=VOISMART
 
@@ -66748,6 +66979,9 @@ OUI:C88B47*
 OUI:C8903E*
  ID_OUI_FROM_DATABASE=Pakton Technologies
 
+OUI:C891F9*
+ ID_OUI_FROM_DATABASE=SAGEMCOM
+
 OUI:C89346*
  ID_OUI_FROM_DATABASE=MXCHIP Company Limited
 
@@ -67000,6 +67234,9 @@ OUI:CC3A61*
 OUI:CC3C3F*
  ID_OUI_FROM_DATABASE=SA.S.S. Datentechnik AG
 
+OUI:CC3D82*
+ ID_OUI_FROM_DATABASE=Intel Corporate
+
 OUI:CC3E5F*
  ID_OUI_FROM_DATABASE=Hewlett Packard
 
@@ -67303,6 +67540,9 @@ OUI:D03761*
 OUI:D03972*
  ID_OUI_FROM_DATABASE=Texas Instruments
 
+OUI:D039B3*
+ ID_OUI_FROM_DATABASE=ARRIS Group, Inc.
+
 OUI:D046DC*
  ID_OUI_FROM_DATABASE=Southwest Research Institute
 
@@ -67735,6 +67975,9 @@ OUI:D48FAA*
 OUI:D491AF*
  ID_OUI_FROM_DATABASE=Electroacustica General Iberica, S.A.
 
+OUI:D49398*
+ ID_OUI_FROM_DATABASE=Nokia Corporation
+
 OUI:D493A0*
  ID_OUI_FROM_DATABASE=Fidelix Oy
 
@@ -68057,7 +68300,7 @@ OUI:D89695*
  ID_OUI_FROM_DATABASE=Apple
 
 OUI:D8973B*
- ID_OUI_FROM_DATABASE=Emerson Network Power Embedded Power
+ ID_OUI_FROM_DATABASE=Artesyn Embedded Technologies
 
 OUI:D89760*
  ID_OUI_FROM_DATABASE=C2 Development, Inc.
@@ -68170,6 +68413,9 @@ OUI:D8DF0D*
 OUI:D8E3AE*
  ID_OUI_FROM_DATABASE=CIRTEC MEDICAL SYSTEMS
 
+OUI:D8E56D*
+ ID_OUI_FROM_DATABASE=TCT Mobile Limited
+
 OUI:D8E72B*
  ID_OUI_FROM_DATABASE=OnPATH Technologies
 
@@ -69319,6 +69565,9 @@ OUI:E8F928*
 OUI:E8FC60*
  ID_OUI_FROM_DATABASE=ELCOM Innovations Private Limited
 
+OUI:E8FCAF*
+ ID_OUI_FROM_DATABASE=NETGEAR INC.,
+
 OUI:EC0ED6*
  ID_OUI_FROM_DATABASE=ITECH INSTRUMENTS SAS
 
@@ -69442,6 +69691,9 @@ OUI:EC888F*
 OUI:EC89F5*
  ID_OUI_FROM_DATABASE=Lenovo Mobile Communication Technology Ltd.
 
+OUI:EC8A4C*
+ ID_OUI_FROM_DATABASE=zte corporation
+
 OUI:EC8EAD*
  ID_OUI_FROM_DATABASE=DLX
 
@@ -70099,6 +70351,9 @@ OUI:F4CE46*
 OUI:F4CFE2*
  ID_OUI_FROM_DATABASE=Cisco
 
+OUI:F4D261*
+ ID_OUI_FROM_DATABASE=SEMOCON Co., Ltd
+
 OUI:F4D9FB*
  ID_OUI_FROM_DATABASE=Samsung Electronics CO., LTD
 
@@ -70111,6 +70366,9 @@ OUI:F4DCDA*
 OUI:F4DCF9*
  ID_OUI_FROM_DATABASE=Huawei Technologies Co., Ltd
 
+OUI:F4DD9E*
+ ID_OUI_FROM_DATABASE=GoPro
+
 OUI:F4E142*
  ID_OUI_FROM_DATABASE=Delta Elektronika BV
 
@@ -70318,6 +70576,9 @@ OUI:F87B8C*
 OUI:F8811A*
  ID_OUI_FROM_DATABASE=OVERKIZ
 
+OUI:F884F2*
+ ID_OUI_FROM_DATABASE=Samsung Electronics Co.,Ltd
+
 OUI:F88C1C*
  ID_OUI_FROM_DATABASE=KAISHUN ELECTRONIC TECHNOLOGY CO., LTD. BEIJING
 
@@ -70586,7 +70847,7 @@ OUI:FC3FAB*
  ID_OUI_FROM_DATABASE=Henan Lanxin Technology Co., Ltd
 
 OUI:FC4463*
- ID_OUI_FROM_DATABASE=Universal Audio
+ ID_OUI_FROM_DATABASE=Universal Audio, Inc
 
 OUI:FC4499*
  ID_OUI_FROM_DATABASE=Swarco LEA d.o.o.
diff --git a/hwdb/20-pci-vendor-model.hwdb b/hwdb/20-pci-vendor-model.hwdb
index 371a685..d54cb2a 100644
--- a/hwdb/20-pci-vendor-model.hwdb
+++ b/hwdb/20-pci-vendor-model.hwdb
@@ -497,6 +497,9 @@ pci:v00000E11d0000F130*
 pci:v00000E11d0000F150*
  ID_MODEL_FROM_DATABASE=NetFlex-3/P ThunderLAN 2.3
 
+pci:v00000E55*
+ ID_VENDOR_FROM_DATABASE=HaSoTec GmbH
+
 pci:v00000EAC*
  ID_VENDOR_FROM_DATABASE=SHF Communication Technologies AG
 
@@ -1169,6 +1172,12 @@ pci:v00001000d00000073sv00001137sd000000B0*
 pci:v00001000d00000073sv00001137sd000000B1*
  ID_MODEL_FROM_DATABASE=UCSC RAID SAS 2008M-8i
 
+pci:v00001000d00000073sv00001137sd000000C2*
+ ID_MODEL_FROM_DATABASE=UCS E-Series Double Wide
+
+pci:v00001000d00000073sv00001137sd000000C3*
+ ID_MODEL_FROM_DATABASE=UCS E-Series Single Wide
+
 pci:v00001000d00000073sv000015D9sd00000400*
  ID_MODEL_FROM_DATABASE=Supermicro SMC2008-iMR
 
@@ -1665,25 +1674,25 @@ pci:v00001002d00001307*
  ID_MODEL_FROM_DATABASE=Kaveri
 
 pci:v00001002d00001309*
- ID_MODEL_FROM_DATABASE=Kaveri
+ ID_MODEL_FROM_DATABASE=Kaveri [Radeon R7 Graphics]
 
 pci:v00001002d0000130A*
- ID_MODEL_FROM_DATABASE=Kaveri
+ ID_MODEL_FROM_DATABASE=Kaveri [Radeon R6 Graphics]
 
 pci:v00001002d0000130B*
- ID_MODEL_FROM_DATABASE=Kaveri
+ ID_MODEL_FROM_DATABASE=Kaveri [Radeon R4 Graphics]
 
 pci:v00001002d0000130C*
- ID_MODEL_FROM_DATABASE=Kaveri
+ ID_MODEL_FROM_DATABASE=Kaveri [Radeon R7 Graphics]
 
 pci:v00001002d0000130D*
- ID_MODEL_FROM_DATABASE=Kaveri
+ ID_MODEL_FROM_DATABASE=Kaveri [Radeon R6 Graphics]
 
 pci:v00001002d0000130E*
- ID_MODEL_FROM_DATABASE=Kaveri
+ ID_MODEL_FROM_DATABASE=Kaveri [Radeon R5 Graphics]
 
 pci:v00001002d0000130F*
- ID_MODEL_FROM_DATABASE=Kaveri [Radeon R7 200 Series]
+ ID_MODEL_FROM_DATABASE=Kaveri [Radeon R7 Graphics]
 
 pci:v00001002d00001310*
  ID_MODEL_FROM_DATABASE=Kaveri
@@ -1695,7 +1704,7 @@ pci:v00001002d00001312*
  ID_MODEL_FROM_DATABASE=Kaveri
 
 pci:v00001002d00001313*
- ID_MODEL_FROM_DATABASE=Kaveri [Radeon R7 200 Series]
+ ID_MODEL_FROM_DATABASE=Kaveri [Radeon R7 Graphics]
 
 pci:v00001002d00001314*
  ID_MODEL_FROM_DATABASE=Wrestler HDMI Audio
@@ -1704,22 +1713,25 @@ pci:v00001002d00001314sv0000174Bsd00001001*
  ID_MODEL_FROM_DATABASE=PURE Fusion Mini
 
 pci:v00001002d00001315*
- ID_MODEL_FROM_DATABASE=Kaveri
+ ID_MODEL_FROM_DATABASE=Kaveri [Radeon R5 Graphics]
 
 pci:v00001002d00001316*
- ID_MODEL_FROM_DATABASE=Kaveri [Radeon R5 200 Series]
+ ID_MODEL_FROM_DATABASE=Kaveri [Radeon R5 Graphics]
 
 pci:v00001002d00001317*
  ID_MODEL_FROM_DATABASE=Kaveri
 
+pci:v00001002d00001318*
+ ID_MODEL_FROM_DATABASE=Kaveri [Radeon R5 Graphics]
+
 pci:v00001002d0000131B*
- ID_MODEL_FROM_DATABASE=Kaveri
+ ID_MODEL_FROM_DATABASE=Kaveri [Radeon R4 Graphics]
 
 pci:v00001002d0000131C*
- ID_MODEL_FROM_DATABASE=Kaveri
+ ID_MODEL_FROM_DATABASE=Kaveri [Radeon R7 Graphics]
 
 pci:v00001002d0000131D*
- ID_MODEL_FROM_DATABASE=Kaveri
+ ID_MODEL_FROM_DATABASE=Kaveri [Radeon R6 Graphics]
 
 pci:v00001002d00001714*
  ID_MODEL_FROM_DATABASE=BeaverCreek HDMI Audio [Radeon HD 6500D and 6400G-6600G series]
@@ -4275,7 +4287,7 @@ pci:v00001002d00006641*
  ID_MODEL_FROM_DATABASE=Saturn PRO [Radeon HD 8930M]
 
 pci:v00001002d00006649*
- ID_MODEL_FROM_DATABASE=Bonaire
+ ID_MODEL_FROM_DATABASE=Bonaire [FirePro W5100]
 
 pci:v00001002d00006650*
  ID_MODEL_FROM_DATABASE=Bonaire
@@ -4326,13 +4338,13 @@ pci:v00001002d00006663sv00001025sd00000846*
  ID_MODEL_FROM_DATABASE=Radeon HD 8570A
 
 pci:v00001002d00006664*
- ID_MODEL_FROM_DATABASE=Jet XT [Radeon R5 M250]
+ ID_MODEL_FROM_DATABASE=Jet XT [Radeon R5 M240]
 
 pci:v00001002d00006665*
  ID_MODEL_FROM_DATABASE=Jet PRO [Radeon R5 M230]
 
 pci:v00001002d00006667*
- ID_MODEL_FROM_DATABASE=Jet ULT [Radeon R5 M240]
+ ID_MODEL_FROM_DATABASE=Jet ULT [Radeon R5 M230]
 
 pci:v00001002d0000666F*
  ID_MODEL_FROM_DATABASE=Sun LE [Radeon HD 8550M]
@@ -5808,7 +5820,7 @@ pci:v00001002d00006792*
  ID_MODEL_FROM_DATABASE=Tahiti
 
 pci:v00001002d00006798*
- ID_MODEL_FROM_DATABASE=Tahiti XT [Radeon HD 7970/R9 280X]
+ ID_MODEL_FROM_DATABASE=Tahiti XT [Radeon HD 7970/8970 OEM / R9 280X]
 
 pci:v00001002d00006798sv00001002sd00003000*
  ID_MODEL_FROM_DATABASE=Tahiti XT2 [Radeon HD 7970 GHz Edition]
@@ -5934,7 +5946,7 @@ pci:v00001002d000067B1*
  ID_MODEL_FROM_DATABASE=Hawaii PRO [Radeon R9 290]
 
 pci:v00001002d000067B9*
- ID_MODEL_FROM_DATABASE=Vesuvius
+ ID_MODEL_FROM_DATABASE=Vesuvius [Radeon R9 295 X2]
 
 pci:v00001002d000067BE*
  ID_MODEL_FROM_DATABASE=Hawaii LE
@@ -5952,7 +5964,7 @@ pci:v00001002d00006800sv00008086sd00002111*
  ID_MODEL_FROM_DATABASE=Radeon HD 7970M
 
 pci:v00001002d00006801*
- ID_MODEL_FROM_DATABASE=Neptune [Radeon HD 8970M]
+ ID_MODEL_FROM_DATABASE=Neptune XT [Radeon HD 8970M]
 
 pci:v00001002d00006801sv00001002sd00000124*
  ID_MODEL_FROM_DATABASE=Radeon HD 8970M
@@ -6017,6 +6029,9 @@ pci:v00001002d00006820*
 pci:v00001002d00006820sv0000103Csd00001851*
  ID_MODEL_FROM_DATABASE=Radeon HD 7750M
 
+pci:v00001002d00006820sv000017AAsd00003801*
+ ID_MODEL_FROM_DATABASE=Radeon R9 M275
+
 pci:v00001002d00006821*
  ID_MODEL_FROM_DATABASE=Venus XT [Radeon HD 8870M]
 
@@ -7817,6 +7832,12 @@ pci:v00001002d000068FAsv000017AFsd00003015*
 pci:v00001002d000068FE*
  ID_MODEL_FROM_DATABASE=Cedar LE
 
+pci:v00001002d00006900*
+ ID_MODEL_FROM_DATABASE=Topaz XT [Radeon R7 M260]
+
+pci:v00001002d00006901*
+ ID_MODEL_FROM_DATABASE=Topaz PRO [Radeon R5 M255]
+
 pci:v00001002d00006920*
  ID_MODEL_FROM_DATABASE=Tonga
 
@@ -10391,6 +10412,15 @@ pci:v00001014d0000034Asv00001014sd00000499*
 pci:v00001014d0000034Asv00001014sd0000049A*
  ID_MODEL_FROM_DATABASE=PCIe3 x16 SAS RAID Internal Adapter 6Gb (57EE)
 
+pci:v00001014d0000034Asv00001014sd000004C7*
+ ID_MODEL_FROM_DATABASE=PCIe3 x 8 Cache SAS RAID Internal Adapter 6GB(2CCA)
+
+pci:v00001014d0000034Asv00001014sd000004C8*
+ ID_MODEL_FROM_DATABASE=PCIe3 x 8 Cache SAS RAID Internal Adapter 6GB(2CD2)
+
+pci:v00001014d0000034Asv00001014sd00000C49*
+ ID_MODEL_FROM_DATABASE=PCIe3 x 8 Cache SAS RAID Internal Adapter 6GB(2CCD)
+
 pci:v00001014d000004AA*
  ID_MODEL_FROM_DATABASE=Flash Adapter 90 (PCIe2 0.9TB)
 
@@ -13793,6 +13823,63 @@ pci:v0000103Cd00003238sv0000103Csd00003211*
 pci:v0000103Cd00003238sv0000103Csd00003212*
  ID_MODEL_FROM_DATABASE=Smart Array E200
 
+pci:v0000103Cd00003239*
+ ID_MODEL_FROM_DATABASE=Smart Array Gen9 Controllers
+
+pci:v0000103Cd00003239sv0000103Csd000021BD*
+ ID_MODEL_FROM_DATABASE=Smart Array
+
+pci:v0000103Cd00003239sv0000103Csd000021BE*
+ ID_MODEL_FROM_DATABASE=Smart Array
+
+pci:v0000103Cd00003239sv0000103Csd000021BF*
+ ID_MODEL_FROM_DATABASE=Smart Array
+
+pci:v0000103Cd00003239sv0000103Csd000021C0*
+ ID_MODEL_FROM_DATABASE=Smart Array
+
+pci:v0000103Cd00003239sv0000103Csd000021C1*
+ ID_MODEL_FROM_DATABASE=Smart Array
+
+pci:v0000103Cd00003239sv0000103Csd000021C2*
+ ID_MODEL_FROM_DATABASE=Smart Array
+
+pci:v0000103Cd00003239sv0000103Csd000021C3*
+ ID_MODEL_FROM_DATABASE=Smart Array
+
+pci:v0000103Cd00003239sv0000103Csd000021C4*
+ ID_MODEL_FROM_DATABASE=Smart Array
+
+pci:v0000103Cd00003239sv0000103Csd000021C5*
+ ID_MODEL_FROM_DATABASE=Smart Array
+
+pci:v0000103Cd00003239sv0000103Csd000021C6*
+ ID_MODEL_FROM_DATABASE=Smart Array
+
+pci:v0000103Cd00003239sv0000103Csd000021C7*
+ ID_MODEL_FROM_DATABASE=Smart Array
+
+pci:v0000103Cd00003239sv0000103Csd000021C8*
+ ID_MODEL_FROM_DATABASE=Smart Array
+
+pci:v0000103Cd00003239sv0000103Csd000021C9*
+ ID_MODEL_FROM_DATABASE=Smart Array
+
+pci:v0000103Cd00003239sv0000103Csd000021CA*
+ ID_MODEL_FROM_DATABASE=Smart Array
+
+pci:v0000103Cd00003239sv0000103Csd000021CB*
+ ID_MODEL_FROM_DATABASE=Smart Array
+
+pci:v0000103Cd00003239sv0000103Csd000021CC*
+ ID_MODEL_FROM_DATABASE=Smart Array
+
+pci:v0000103Cd00003239sv0000103Csd000021CD*
+ ID_MODEL_FROM_DATABASE=Smart Array
+
+pci:v0000103Cd00003239sv0000103Csd000021CE*
+ ID_MODEL_FROM_DATABASE=Smart Array
+
 pci:v0000103Cd0000323A*
  ID_MODEL_FROM_DATABASE=Smart Array G6 controllers
 
@@ -17954,6 +18041,54 @@ pci:v0000109Ed0000036Esv000018ACsd0000D500*
 pci:v0000109Ed0000036Esv0000270Fsd0000FC00*
  ID_MODEL_FROM_DATABASE=Digitop DTT-1000
 
+pci:v0000109Ed0000036Esv0000AA00sd00001460*
+ ID_MODEL_FROM_DATABASE=Spectra8 CardA Input0
+
+pci:v0000109Ed0000036Esv0000AA01sd00001461*
+ ID_MODEL_FROM_DATABASE=Spectra8 CardA Input1
+
+pci:v0000109Ed0000036Esv0000AA02sd00001462*
+ ID_MODEL_FROM_DATABASE=Spectra8 CardA Input2
+
+pci:v0000109Ed0000036Esv0000AA03sd00001463*
+ ID_MODEL_FROM_DATABASE=Spectra8 CardA Input3
+
+pci:v0000109Ed0000036Esv0000AA04sd00001464*
+ ID_MODEL_FROM_DATABASE=Spectra8 CardB Input0
+
+pci:v0000109Ed0000036Esv0000AA05sd00001465*
+ ID_MODEL_FROM_DATABASE=Spectra8 CardB Input1
+
+pci:v0000109Ed0000036Esv0000AA06sd00001466*
+ ID_MODEL_FROM_DATABASE=Spectra8 CardB Input2
+
+pci:v0000109Ed0000036Esv0000AA07sd00001467*
+ ID_MODEL_FROM_DATABASE=Spectra8 CardB Input3
+
+pci:v0000109Ed0000036Esv0000AA08sd00001468*
+ ID_MODEL_FROM_DATABASE=Spectra8 CardC Input0
+
+pci:v0000109Ed0000036Esv0000AA09sd00001469*
+ ID_MODEL_FROM_DATABASE=Spectra8 CardC Input1
+
+pci:v0000109Ed0000036Esv0000AA0Asd0000146A*
+ ID_MODEL_FROM_DATABASE=Spectra8 CardC Input2
+
+pci:v0000109Ed0000036Esv0000AA0Bsd0000146B*
+ ID_MODEL_FROM_DATABASE=Spectra8 CardC Input3
+
+pci:v0000109Ed0000036Esv0000AA0Csd0000146C*
+ ID_MODEL_FROM_DATABASE=Spectra8 CardD Input0
+
+pci:v0000109Ed0000036Esv0000AA0Dsd0000146D*
+ ID_MODEL_FROM_DATABASE=Spectra8 CardD Input1
+
+pci:v0000109Ed0000036Esv0000AA0Esd0000146E*
+ ID_MODEL_FROM_DATABASE=Spectra8 CardD Input2
+
+pci:v0000109Ed0000036Esv0000AA0Fsd0000146F*
+ ID_MODEL_FROM_DATABASE=Spectra8 CardD Input3
+
 pci:v0000109Ed0000036Esv0000BD11sd00001200*
  ID_MODEL_FROM_DATABASE=PCTV pro (TV + FM stereo receiver)
 
@@ -26009,6 +26144,9 @@ pci:v000010DEd00000E1A*
 pci:v000010DEd00000E1B*
  ID_MODEL_FROM_DATABASE=GK107 HDMI Audio Controller
 
+pci:v000010DEd00000E1Bsv0000103Csd0000197B*
+ ID_MODEL_FROM_DATABASE=ZBook 15
+
 pci:v000010DEd00000E1Bsv00001043sd00008428*
  ID_MODEL_FROM_DATABASE=GTX650-DC-1GD5
 
@@ -26165,6 +26303,9 @@ pci:v000010DEd00000FF5*
 pci:v000010DEd00000FF6*
  ID_MODEL_FROM_DATABASE=GK107GLM [Quadro K1100M]
 
+pci:v000010DEd00000FF6sv0000103Csd0000197B*
+ ID_MODEL_FROM_DATABASE=ZBook 15
+
 pci:v000010DEd00000FF7*
  ID_MODEL_FROM_DATABASE=GK107GL [GRID K140Q vGPU]
 
@@ -26234,6 +26375,12 @@ pci:v000010DEd00001005sv00003842sd00002794*
 pci:v000010DEd00001005sv00003842sd00002795*
  ID_MODEL_FROM_DATABASE=GeForce GTX Titan SC Hydro Copper Signature
 
+pci:v000010DEd00001007*
+ ID_MODEL_FROM_DATABASE=GK110 [GeForce GTX 780 Rev. 2]
+
+pci:v000010DEd00001008*
+ ID_MODEL_FROM_DATABASE=GK110 [GeForce GTX 780 Ti Rev. 2]
+
 pci:v000010DEd0000100A*
  ID_MODEL_FROM_DATABASE=GK110B [GeForce GTX 780 Ti]
 
@@ -26294,6 +26441,9 @@ pci:v000010DEd0000104Asv000010B0sd0000104A*
 pci:v000010DEd0000104B*
  ID_MODEL_FROM_DATABASE=GF119 [GeForce GT 625 OEM]
 
+pci:v000010DEd0000104C*
+ ID_MODEL_FROM_DATABASE=GF119 [GeForce GT 705]
+
 pci:v000010DEd00001050*
  ID_MODEL_FROM_DATABASE=GF119M [GeForce GT 520M]
 
@@ -26333,6 +26483,24 @@ pci:v000010DEd00001058sv00001043sd000010BC*
 pci:v000010DEd00001058sv00001043sd00001652*
  ID_MODEL_FROM_DATABASE=GeForce GT 610M
 
+pci:v000010DEd00001058sv000017AAsd0000367A*
+ ID_MODEL_FROM_DATABASE=GeForce 610M
+
+pci:v000010DEd00001058sv000017AAsd00003682*
+ ID_MODEL_FROM_DATABASE=GeForce 800A
+
+pci:v000010DEd00001058sv000017AAsd00003687*
+ ID_MODEL_FROM_DATABASE=GeForce 800A
+
+pci:v000010DEd00001058sv000017AAsd00003692*
+ ID_MODEL_FROM_DATABASE=GeForce 705A
+
+pci:v000010DEd00001058sv000017AAsd00003695*
+ ID_MODEL_FROM_DATABASE=GeForce 800A
+
+pci:v000010DEd00001058sv000017AAsd0000A117*
+ ID_MODEL_FROM_DATABASE=GeForce 610M
+
 pci:v000010DEd00001059*
  ID_MODEL_FROM_DATABASE=GF119M [GeForce 610M]
 
@@ -26351,6 +26519,15 @@ pci:v000010DEd0000105B*
 pci:v000010DEd0000105Bsv0000103Csd00002AFB*
  ID_MODEL_FROM_DATABASE=GeForce 705A
 
+pci:v000010DEd0000105Bsv000017AAsd0000309D*
+ ID_MODEL_FROM_DATABASE=GeForce 705A
+
+pci:v000010DEd0000105Bsv000017AAsd000030B1*
+ ID_MODEL_FROM_DATABASE=GeForce 800A
+
+pci:v000010DEd0000105Bsv000017AAsd000036A1*
+ ID_MODEL_FROM_DATABASE=GeForce 800A
+
 pci:v000010DEd0000107C*
  ID_MODEL_FROM_DATABASE=GF119 [NVS 315]
 
@@ -26675,6 +26852,30 @@ pci:v000010DEd00001140sv00001043sd000014C7*
 pci:v000010DEd00001140sv00001043sd00001507*
  ID_MODEL_FROM_DATABASE=GeForce GT 620M
 
+pci:v000010DEd00001140sv00001043sd000015AD*
+ ID_MODEL_FROM_DATABASE=GeForce 820M
+
+pci:v000010DEd00001140sv00001043sd000015ED*
+ ID_MODEL_FROM_DATABASE=GeForce 820M
+
+pci:v000010DEd00001140sv00001043sd0000160D*
+ ID_MODEL_FROM_DATABASE=GeForce 820M
+
+pci:v000010DEd00001140sv00001043sd0000163D*
+ ID_MODEL_FROM_DATABASE=GeForce 820M
+
+pci:v000010DEd00001140sv00001043sd0000166D*
+ ID_MODEL_FROM_DATABASE=GeForce 820M
+
+pci:v000010DEd00001140sv00001043sd000016CD*
+ ID_MODEL_FROM_DATABASE=GeForce 820M
+
+pci:v000010DEd00001140sv00001043sd000016DD*
+ ID_MODEL_FROM_DATABASE=GeForce 820M
+
+pci:v000010DEd00001140sv00001043sd0000170D*
+ ID_MODEL_FROM_DATABASE=GeForce 820M
+
 pci:v000010DEd00001140sv00001043sd00002132*
  ID_MODEL_FROM_DATABASE=GeForce GT 620M
 
@@ -26699,6 +26900,12 @@ pci:v000010DEd00001140sv00001043sd0000223A*
 pci:v000010DEd00001140sv00001043sd0000224A*
  ID_MODEL_FROM_DATABASE=GeForce GT 710M
 
+pci:v000010DEd00001140sv00001043sd0000227A*
+ ID_MODEL_FROM_DATABASE=GeForce 820M
+
+pci:v000010DEd00001140sv00001043sd0000228A*
+ ID_MODEL_FROM_DATABASE=GeForce 820M
+
 pci:v000010DEd00001140sv00001043sd00008595*
  ID_MODEL_FROM_DATABASE=GeForce GT 720M
 
@@ -26864,9 +27071,48 @@ pci:v000010DEd00001140sv000017AAsd00002213*
 pci:v000010DEd00001140sv000017AAsd00002220*
  ID_MODEL_FROM_DATABASE=GeForce GT 720M
 
+pci:v000010DEd00001140sv000017AAsd0000309C*
+ ID_MODEL_FROM_DATABASE=GeForce GT 720A
+
+pci:v000010DEd00001140sv000017AAsd000030B4*
+ ID_MODEL_FROM_DATABASE=GeForce 820A
+
 pci:v000010DEd00001140sv000017AAsd00003656*
  ID_MODEL_FROM_DATABASE=GeForce GT 620M
 
+pci:v000010DEd00001140sv000017AAsd0000365A*
+ ID_MODEL_FROM_DATABASE=GeForce 705M
+
+pci:v000010DEd00001140sv000017AAsd0000365E*
+ ID_MODEL_FROM_DATABASE=GeForce 800M
+
+pci:v000010DEd00001140sv000017AAsd0000366C*
+ ID_MODEL_FROM_DATABASE=GeForce 800M
+
+pci:v000010DEd00001140sv000017AAsd00003685*
+ ID_MODEL_FROM_DATABASE=GeForce 800M
+
+pci:v000010DEd00001140sv000017AAsd00003686*
+ ID_MODEL_FROM_DATABASE=GeForce 800M
+
+pci:v000010DEd00001140sv000017AAsd00003687*
+ ID_MODEL_FROM_DATABASE=GeForce 705A
+
+pci:v000010DEd00001140sv000017AAsd00003696*
+ ID_MODEL_FROM_DATABASE=GeForce 820A
+
+pci:v000010DEd00001140sv000017AAsd0000369B*
+ ID_MODEL_FROM_DATABASE=GeForce 820A
+
+pci:v000010DEd00001140sv000017AAsd0000369C*
+ ID_MODEL_FROM_DATABASE=GeForce 820A
+
+pci:v000010DEd00001140sv000017AAsd0000369D*
+ ID_MODEL_FROM_DATABASE=GeForce 820A
+
+pci:v000010DEd00001140sv000017AAsd0000369E*
+ ID_MODEL_FROM_DATABASE=GeForce 820A
+
 pci:v000010DEd00001140sv000017AAsd00003800*
  ID_MODEL_FROM_DATABASE=GeForce GT 720M
 
@@ -27044,6 +27290,9 @@ pci:v000010DEd0000118F*
 pci:v000010DEd00001193*
  ID_MODEL_FROM_DATABASE=GK104 [GeForce GTX 760 Ti OEM]
 
+pci:v000010DEd00001195*
+ ID_MODEL_FROM_DATABASE=GK104 [GeForce GTX 660 Rev. 2]
+
 pci:v000010DEd00001198*
  ID_MODEL_FROM_DATABASE=GK104M [GeForce GTX 880M]
 
@@ -27173,6 +27422,9 @@ pci:v000010DEd000011E2*
 pci:v000010DEd000011E3*
  ID_MODEL_FROM_DATABASE=GK106M [GeForce GTX 760M]
 
+pci:v000010DEd000011E3sv000017AAsd00003683*
+ ID_MODEL_FROM_DATABASE=GeForce GTX 760A
+
 pci:v000010DEd000011FA*
  ID_MODEL_FROM_DATABASE=GK106GL [Quadro K4000]
 
@@ -27323,6 +27575,9 @@ pci:v000010DEd00001282*
 pci:v000010DEd00001284*
  ID_MODEL_FROM_DATABASE=GK208 [GeForce GT 630 Rev. 2]
 
+pci:v000010DEd00001286*
+ ID_MODEL_FROM_DATABASE=GK208 [GeForce GT 720]
+
 pci:v000010DEd00001290*
  ID_MODEL_FROM_DATABASE=GK208M [GeForce GT 730M]
 
@@ -27380,6 +27635,15 @@ pci:v000010DEd00001340*
 pci:v000010DEd00001341*
  ID_MODEL_FROM_DATABASE=GM108M [GeForce 840M]
 
+pci:v000010DEd00001341sv000017AAsd00003697*
+ ID_MODEL_FROM_DATABASE=GeForce 840A
+
+pci:v000010DEd00001341sv000017AAsd00003699*
+ ID_MODEL_FROM_DATABASE=GeForce 840A
+
+pci:v000010DEd00001341sv000017AAsd0000369C*
+ ID_MODEL_FROM_DATABASE=GeForce 840A
+
 pci:v000010DEd00001380*
  ID_MODEL_FROM_DATABASE=GM107 [GeForce GTX 750 Ti]
 
@@ -27395,6 +27659,12 @@ pci:v000010DEd00001390*
 pci:v000010DEd00001391*
  ID_MODEL_FROM_DATABASE=GM107M [GeForce GTX 850M]
 
+pci:v000010DEd00001391sv000017AAsd00003697*
+ ID_MODEL_FROM_DATABASE=GeForce GTX 850A
+
+pci:v000010DEd00001391sv000017AAsd0000A125*
+ ID_MODEL_FROM_DATABASE=GeForce GTX 850A
+
 pci:v000010DEd00001392*
  ID_MODEL_FROM_DATABASE=GM107M [GeForce GTX 860M]
 
@@ -27839,6 +28109,9 @@ pci:v000010ECd00005227*
 pci:v000010ECd00005227sv000017AAsd0000220E*
  ID_MODEL_FROM_DATABASE=ThinkPad T440p
 
+pci:v000010ECd00005227sv000017AAsd00002214*
+ ID_MODEL_FROM_DATABASE=ThinkPad X240
+
 pci:v000010ECd00005229*
  ID_MODEL_FROM_DATABASE=RTS5229 PCI Express Card Reader
 
@@ -27848,6 +28121,12 @@ pci:v000010ECd00005229sv00001025sd00000813*
 pci:v000010ECd00005229sv0000103Csd0000194E*
  ID_MODEL_FROM_DATABASE=ProBook 455 G1 Notebook
 
+pci:v000010ECd00005249*
+ ID_MODEL_FROM_DATABASE=RTS5249 PCI Express Card Reader
+
+pci:v000010ECd00005249sv0000103Csd00001909*
+ ID_MODEL_FROM_DATABASE=ZBook 15
+
 pci:v000010ECd00005288*
  ID_MODEL_FROM_DATABASE=Barossa PCI Express Card Reader
 
@@ -30561,7 +30840,7 @@ pci:v0000110Ad00004021*
  ID_MODEL_FROM_DATABASE=SIMATIC NET CP 5512 (Profibus and MPI Cardbus Adapter)
 
 pci:v0000110Ad00004029*
- ID_MODEL_FROM_DATABASE=SIMATIC NET CP 5614 A2
+ ID_MODEL_FROM_DATABASE=SIMATIC NET CP 5613 A2
 
 pci:v0000110Ad00004029sv0000110Asd00004029*
  ID_MODEL_FROM_DATABASE=SIMATIC NET CP 5613 A2
@@ -30581,6 +30860,9 @@ pci:v0000110Ad00004038*
 pci:v0000110Ad00004069*
  ID_MODEL_FROM_DATABASE=SIMATIC NET CP 5623
 
+pci:v0000110Ad00004069sv0000110Asd00004069*
+ ID_MODEL_FROM_DATABASE=SIMATIC NET CP 5623
+
 pci:v0000110Ad00004069sv0000110Asd0000C069*
  ID_MODEL_FROM_DATABASE=SIMATIC NET CP 5624
 
@@ -31541,6 +31823,9 @@ pci:v00001131d00007133sv0000153Bsd00001160*
 pci:v00001131d00007133sv0000153Bsd00001162*
  ID_MODEL_FROM_DATABASE=Terratec Cinergy 400 mobile
 
+pci:v00001131d00007133sv000017DEsd00007256*
+ ID_MODEL_FROM_DATABASE=PlusTV All In One PI610 card
+
 pci:v00001131d00007133sv000017DEsd00007350*
  ID_MODEL_FROM_DATABASE=ATSC 110 Digital / Analog HDTV Tuner
 
@@ -33651,7 +33936,7 @@ pci:v00001180d00000476sv000014EFsd00000220*
  ID_MODEL_FROM_DATABASE=PCD-RP-220S
 
 pci:v00001180d00000476sv000017AAsd0000201C*
- ID_MODEL_FROM_DATABASE=ThinkPad X60s
+ ID_MODEL_FROM_DATABASE=ThinkPad X60/X60s
 
 pci:v00001180d00000476sv000017AAsd000020C4*
  ID_MODEL_FROM_DATABASE=ThinkPad T61
@@ -33711,7 +33996,7 @@ pci:v00001180d00000552sv0000144Dsd0000C00C*
  ID_MODEL_FROM_DATABASE=P30/P35 notebook
 
 pci:v00001180d00000552sv000017AAsd0000201E*
- ID_MODEL_FROM_DATABASE=ThinkPad X60s
+ ID_MODEL_FROM_DATABASE=ThinkPad X60/X60s
 
 pci:v00001180d00000554*
  ID_MODEL_FROM_DATABASE=R5C554
@@ -33813,7 +34098,7 @@ pci:v00001180d00000822sv0000144Dsd0000C018*
  ID_MODEL_FROM_DATABASE=X20 IV
 
 pci:v00001180d00000822sv000017AAsd0000201D*
- ID_MODEL_FROM_DATABASE=ThinkPad X60s
+ ID_MODEL_FROM_DATABASE=ThinkPad X60/X60s
 
 pci:v00001180d00000822sv000017AAsd000020C7*
  ID_MODEL_FROM_DATABASE=ThinkPad T61
@@ -33833,6 +34118,9 @@ pci:v00001180d00000832sv00001028sd000001D7*
 pci:v00001180d00000832sv00001028sd000001F3*
  ID_MODEL_FROM_DATABASE=Inspiron 1420
 
+pci:v00001180d00000832sv00001028sd0000024D*
+ ID_MODEL_FROM_DATABASE=Latitude E4300
+
 pci:v00001180d00000832sv0000103Csd000030B5*
  ID_MODEL_FROM_DATABASE=Presario V3242AU
 
@@ -49442,6 +49730,18 @@ pci:v00001797d00006812*
 pci:v00001797d00006813*
  ID_MODEL_FROM_DATABASE=TW6816 multimedia video controller
 
+pci:v00001797d00006814*
+ ID_MODEL_FROM_DATABASE=TW6816 multimedia video controller
+
+pci:v00001797d00006815*
+ ID_MODEL_FROM_DATABASE=TW6816 multimedia video controller
+
+pci:v00001797d00006816*
+ ID_MODEL_FROM_DATABASE=TW6816 multimedia video controller
+
+pci:v00001797d00006817*
+ ID_MODEL_FROM_DATABASE=TW6816 multimedia video controller
+
 pci:v00001799*
  ID_VENDOR_FROM_DATABASE=Belkin
 
@@ -50978,12 +51278,33 @@ pci:v00001924d00000813sv00001924sd00007904*
 pci:v00001924d00000903*
  ID_MODEL_FROM_DATABASE=SFC9120
 
+pci:v00001924d00000903sv00001014sd000004CC*
+ ID_MODEL_FROM_DATABASE=SFN7122F-R2 2x10GbE SFP+ Flareon Ultra
+
 pci:v00001924d00000903sv00001924sd00008002*
  ID_MODEL_FROM_DATABASE=SFN7122F-R1 SFP+ Server Adapter
 
+pci:v00001924d00000903sv00001924sd00008003*
+ ID_MODEL_FROM_DATABASE=SFN7x41Q-R1 Flareon Ultra 7000 Series 10/40G Adapter
+
 pci:v00001924d00000903sv00001924sd00008006*
  ID_MODEL_FROM_DATABASE=SFN7022F-R1 SFP+ Server Adapter
 
+pci:v00001924d00000903sv00001924sd00008007*
+ ID_MODEL_FROM_DATABASE=SFN7322F-R2 Precision Time SFP+ Server Adapter
+
+pci:v00001924d00000903sv00001924sd00008009*
+ ID_MODEL_FROM_DATABASE=SFN7x22F-R2 Flareon Ultra 7000 Series 10G Adapter
+
+pci:v00001924d00000903sv00001924sd0000800A*
+ ID_MODEL_FROM_DATABASE=SFN7x02F-R2 Flareon 7000 Series 10G Adapter
+
+pci:v00001924d00000923*
+ ID_MODEL_FROM_DATABASE=SFC9140
+
+pci:v00001924d00000923sv00001924sd0000800B*
+ ID_MODEL_FROM_DATABASE=SFN7x42Q-R1 Flareon Ultra 7000 Series 10/40G Adapter
+
 pci:v00001924d00001803*
  ID_MODEL_FROM_DATABASE=SFC9020 Virtual Function [Solarstorm]
 
@@ -51227,6 +51548,9 @@ pci:v00001957d00000083*
 pci:v00001957d00000084*
  ID_MODEL_FROM_DATABASE=MPC8347E PBGA
 
+pci:v00001957d00000084sv0000110Asd00004074*
+ ID_MODEL_FROM_DATABASE=SIMATIC NET CP 1628
+
 pci:v00001957d00000085*
  ID_MODEL_FROM_DATABASE=MPC8347 PBGA
 
@@ -51456,7 +51780,7 @@ pci:v00001969d00002062*
  ID_MODEL_FROM_DATABASE=AR8152 v2.0 Fast Ethernet
 
 pci:v00001969d0000E091*
- ID_MODEL_FROM_DATABASE=Killer E2200 Gigabit Ethernet Controller
+ ID_MODEL_FROM_DATABASE=Killer E220x Gigabit Ethernet Controller
 
 pci:v0000196A*
  ID_VENDOR_FROM_DATABASE=Sensory Networks Inc.
@@ -52214,8 +52538,14 @@ pci:v00001AF4d00001005*
 pci:v00001AF4d00001009*
  ID_MODEL_FROM_DATABASE=Virtio filesystem
 
+pci:v00001AF4d00001010*
+ ID_MODEL_FROM_DATABASE=Virtio GPU
+
+pci:v00001AF4d00001012*
+ ID_MODEL_FROM_DATABASE=Virtio input device
+
 pci:v00001AF4d00001110*
- ID_MODEL_FROM_DATABASE=Virtio Inter-VM shared memory
+ ID_MODEL_FROM_DATABASE=Inter-VM shared memory
 
 pci:v00001AF4d00001110sv00001AF4sd00001100*
  ID_MODEL_FROM_DATABASE=QEMU Virtual Machine
@@ -52397,6 +52727,9 @@ pci:v00001B4Bd000091A4*
 pci:v00001B4Bd00009230*
  ID_MODEL_FROM_DATABASE=88SE9230 PCIe SATA 6Gb/s Controller
 
+pci:v00001B4Bd00009445*
+ ID_MODEL_FROM_DATABASE=88SE9445 PCIe 2.0 x4 4-Port SAS/SATA 6 Gbps RAID Controller
+
 pci:v00001B4Bd00009480*
  ID_MODEL_FROM_DATABASE=88SE9480 SAS/SATA 6Gb/s RAID controller
 
@@ -55550,15 +55883,24 @@ pci:v00008086d00000964*
 pci:v00008086d00000A04*
  ID_MODEL_FROM_DATABASE=Haswell-ULT DRAM Controller
 
+pci:v00008086d00000A04sv000017AAsd00002214*
+ ID_MODEL_FROM_DATABASE=ThinkPad X240
+
 pci:v00008086d00000A06*
  ID_MODEL_FROM_DATABASE=Haswell-ULT Integrated Graphics Controller
 
 pci:v00008086d00000A0C*
  ID_MODEL_FROM_DATABASE=Haswell-ULT HD Audio Controller
 
+pci:v00008086d00000A0Csv000017AAsd00002214*
+ ID_MODEL_FROM_DATABASE=ThinkPad X240
+
 pci:v00008086d00000A16*
  ID_MODEL_FROM_DATABASE=Haswell-ULT Integrated Graphics Controller
 
+pci:v00008086d00000A16sv000017AAsd00002214*
+ ID_MODEL_FROM_DATABASE=ThinkPad X240
+
 pci:v00008086d00000A22*
  ID_MODEL_FROM_DATABASE=Haswell-ULT Integrated Graphics Controller
 
@@ -55655,6 +55997,9 @@ pci:v00008086d00000C01*
 pci:v00008086d00000C04*
  ID_MODEL_FROM_DATABASE=Xeon E3-1200 v3/4th Gen Core Processor DRAM Controller
 
+pci:v00008086d00000C04sv0000103Csd00001909*
+ ID_MODEL_FROM_DATABASE=ZBook 15
+
 pci:v00008086d00000C04sv000017AAsd0000220E*
  ID_MODEL_FROM_DATABASE=ThinkPad T440p
 
@@ -57360,7 +57705,7 @@ pci:v00008086d0000109Asv000017AAsd00002001*
  ID_MODEL_FROM_DATABASE=ThinkPad T60
 
 pci:v00008086d0000109Asv000017AAsd0000207E*
- ID_MODEL_FROM_DATABASE=ThinkPad X60s
+ ID_MODEL_FROM_DATABASE=ThinkPad X60/X60s
 
 pci:v00008086d0000109Asv00008086sd0000109A*
  ID_MODEL_FROM_DATABASE=PRO/1000 PL Network Connection
@@ -58847,6 +59192,9 @@ pci:v00008086d00001539*
 pci:v00008086d0000153A*
  ID_MODEL_FROM_DATABASE=Ethernet Connection I217-LM
 
+pci:v00008086d0000153Asv0000103Csd00001909*
+ ID_MODEL_FROM_DATABASE=ZBook 15
+
 pci:v00008086d0000153Asv000017AAsd0000220E*
  ID_MODEL_FROM_DATABASE=ThinkPad T440p
 
@@ -58901,6 +59249,9 @@ pci:v00008086d00001559*
 pci:v00008086d0000155A*
  ID_MODEL_FROM_DATABASE=Ethernet Connection I218-LM
 
+pci:v00008086d0000155Asv000017AAsd00002214*
+ ID_MODEL_FROM_DATABASE=ThinkPad X240
+
 pci:v00008086d0000155C*
  ID_MODEL_FROM_DATABASE=Ethernet Server Bypass Adapter
 
@@ -63816,7 +64167,7 @@ pci:v00008086d000027A0sv000017AAsd00002015*
  ID_MODEL_FROM_DATABASE=ThinkPad T60
 
 pci:v00008086d000027A0sv000017AAsd00002017*
- ID_MODEL_FROM_DATABASE=ThinkPad T60/R60 series
+ ID_MODEL_FROM_DATABASE=ThinkPad R60/T60/X60 series
 
 pci:v00008086d000027A1*
  ID_MODEL_FROM_DATABASE=Mobile 945GM/PM/GMS, 943/940GML and 945GT Express PCI Express Root Port
@@ -63834,7 +64185,7 @@ pci:v00008086d000027A2sv0000103Csd000030A1*
  ID_MODEL_FROM_DATABASE=NC2400
 
 pci:v00008086d000027A2sv000017AAsd0000201A*
- ID_MODEL_FROM_DATABASE=ThinkPad T60/R60 series
+ ID_MODEL_FROM_DATABASE=ThinkPad R60/T60/X60 series
 
 pci:v00008086d000027A2sv00009902sd00001584*
  ID_MODEL_FROM_DATABASE=CCE MPL-D10H120F
@@ -63849,7 +64200,7 @@ pci:v00008086d000027A6sv00001775sd000011CC*
  ID_MODEL_FROM_DATABASE=CC11/CL11 integrated graphics (secondary)
 
 pci:v00008086d000027A6sv000017AAsd0000201A*
- ID_MODEL_FROM_DATABASE=ThinkPad T60/R60 series
+ ID_MODEL_FROM_DATABASE=ThinkPad R60/T60/X60 series
 
 pci:v00008086d000027AC*
  ID_MODEL_FROM_DATABASE=Mobile 945GSE Express Memory Controller Hub
@@ -63915,7 +64266,7 @@ pci:v00008086d000027B9sv000010F7sd00008338*
  ID_MODEL_FROM_DATABASE=Panasonic CF-Y5 laptop
 
 pci:v00008086d000027B9sv000017AAsd00002009*
- ID_MODEL_FROM_DATABASE=ThinkPad T60/R60 series
+ ID_MODEL_FROM_DATABASE=ThinkPad R60/T60/X60 series
 
 pci:v00008086d000027BC*
  ID_MODEL_FROM_DATABASE=NM10 Family LPC Controller
@@ -64035,7 +64386,7 @@ pci:v00008086d000027C5sv0000103Csd000030A3*
  ID_MODEL_FROM_DATABASE=Compaq nw8440
 
 pci:v00008086d000027C5sv000017AAsd0000200D*
- ID_MODEL_FROM_DATABASE=ThinkPad T60/R60 series
+ ID_MODEL_FROM_DATABASE=ThinkPad R60/T60/X60 series
 
 pci:v00008086d000027C6*
  ID_MODEL_FROM_DATABASE=82801GHM (ICH7-M DH) SATA Controller [RAID mode]
@@ -64095,7 +64446,7 @@ pci:v00008086d000027C8sv00001775sd000011CC*
  ID_MODEL_FROM_DATABASE=CC11/CL11
 
 pci:v00008086d000027C8sv000017AAsd0000200A*
- ID_MODEL_FROM_DATABASE=ThinkPad T60/R60 series
+ ID_MODEL_FROM_DATABASE=ThinkPad R60/T60/X60 series
 
 pci:v00008086d000027C8sv00008086sd00004F4D*
  ID_MODEL_FROM_DATABASE=DeskTop Board D510MO
@@ -64161,7 +64512,7 @@ pci:v00008086d000027C9sv00001775sd000011CC*
  ID_MODEL_FROM_DATABASE=CC11/CL11
 
 pci:v00008086d000027C9sv000017AAsd0000200A*
- ID_MODEL_FROM_DATABASE=ThinkPad T60/R60 series
+ ID_MODEL_FROM_DATABASE=ThinkPad R60/T60/X60 series
 
 pci:v00008086d000027C9sv00008086sd00004F4D*
  ID_MODEL_FROM_DATABASE=DeskTop Board D510MO
@@ -64227,7 +64578,7 @@ pci:v00008086d000027CAsv00001775sd000011CC*
  ID_MODEL_FROM_DATABASE=CC11/CL11
 
 pci:v00008086d000027CAsv000017AAsd0000200A*
- ID_MODEL_FROM_DATABASE=ThinkPad T60/R60 series
+ ID_MODEL_FROM_DATABASE=ThinkPad R60/T60/X60 series
 
 pci:v00008086d000027CAsv00008086sd00004F4D*
  ID_MODEL_FROM_DATABASE=DeskTop Board D510MO
@@ -64287,7 +64638,7 @@ pci:v00008086d000027CBsv00001775sd000011CC*
  ID_MODEL_FROM_DATABASE=CC11/CL11
 
 pci:v00008086d000027CBsv000017AAsd0000200A*
- ID_MODEL_FROM_DATABASE=ThinkPad T60/R60 series
+ ID_MODEL_FROM_DATABASE=ThinkPad R60/T60/X60 series
 
 pci:v00008086d000027CBsv00008086sd00004F4D*
  ID_MODEL_FROM_DATABASE=DeskTop Board D510MO
@@ -64347,7 +64698,7 @@ pci:v00008086d000027CCsv00001775sd000011CC*
  ID_MODEL_FROM_DATABASE=CC11/CL11
 
 pci:v00008086d000027CCsv000017AAsd0000200B*
- ID_MODEL_FROM_DATABASE=ThinkPad T60/R60 series
+ ID_MODEL_FROM_DATABASE=ThinkPad R60/T60/X60 series
 
 pci:v00008086d000027CCsv00008086sd00004F4D*
  ID_MODEL_FROM_DATABASE=DeskTop Board D510MO
@@ -64509,7 +64860,7 @@ pci:v00008086d000027D8sv00001734sd000010AD*
  ID_MODEL_FROM_DATABASE=Conexant softmodem SmartCP
 
 pci:v00008086d000027D8sv000017AAsd00002010*
- ID_MODEL_FROM_DATABASE=ThinkPad T60/R60 series
+ ID_MODEL_FROM_DATABASE=ThinkPad R60/T60/X60 series
 
 pci:v00008086d000027D8sv000017AAsd00003802*
  ID_MODEL_FROM_DATABASE=Lenovo 3000 C200 audio [Realtek ALC861VD]
@@ -64569,7 +64920,7 @@ pci:v00008086d000027DAsv00001775sd000011CC*
  ID_MODEL_FROM_DATABASE=CC11/CL11
 
 pci:v00008086d000027DAsv000017AAsd0000200F*
- ID_MODEL_FROM_DATABASE=ThinkPad T60/R60 series
+ ID_MODEL_FROM_DATABASE=ThinkPad R60/T60/X60 series
 
 pci:v00008086d000027DAsv00008086sd00004F4D*
  ID_MODEL_FROM_DATABASE=DeskTop Board D510MO
@@ -64647,7 +64998,7 @@ pci:v00008086d000027DFsv00001775sd000011CC*
  ID_MODEL_FROM_DATABASE=CC11/CL11
 
 pci:v00008086d000027DFsv000017AAsd0000200C*
- ID_MODEL_FROM_DATABASE=ThinkPad T60/R60 series
+ ID_MODEL_FROM_DATABASE=ThinkPad R60/T60/X60 series
 
 pci:v00008086d000027DFsv00008086sd0000544E*
  ID_MODEL_FROM_DATABASE=DeskTop Board D945GTP
@@ -70358,6 +70709,9 @@ pci:v00008086d00008C02*
 pci:v00008086d00008C03*
  ID_MODEL_FROM_DATABASE=8 Series/C220 Series Chipset Family 6-port SATA Controller 1 [AHCI mode]
 
+pci:v00008086d00008C03sv0000103Csd00001909*
+ ID_MODEL_FROM_DATABASE=ZBook 15
+
 pci:v00008086d00008C03sv000017AAsd0000220E*
  ID_MODEL_FROM_DATABASE=ThinkPad T440p
 
@@ -70442,6 +70796,9 @@ pci:v00008086d00008C1F*
 pci:v00008086d00008C20*
  ID_MODEL_FROM_DATABASE=8 Series/C220 Series Chipset High Definition Audio Controller
 
+pci:v00008086d00008C20sv0000103Csd00001909*
+ ID_MODEL_FROM_DATABASE=ZBook 15
+
 pci:v00008086d00008C20sv000017AAsd0000220E*
  ID_MODEL_FROM_DATABASE=ThinkPad T440p
 
@@ -70451,6 +70808,9 @@ pci:v00008086d00008C21*
 pci:v00008086d00008C22*
  ID_MODEL_FROM_DATABASE=8 Series/C220 Series Chipset Family SMBus Controller
 
+pci:v00008086d00008C22sv0000103Csd00001909*
+ ID_MODEL_FROM_DATABASE=ZBook 15
+
 pci:v00008086d00008C22sv000017AAsd0000220E*
  ID_MODEL_FROM_DATABASE=ThinkPad T440p
 
@@ -70463,18 +70823,27 @@ pci:v00008086d00008C24*
 pci:v00008086d00008C26*
  ID_MODEL_FROM_DATABASE=8 Series/C220 Series Chipset Family USB EHCI #1
 
+pci:v00008086d00008C26sv0000103Csd00001909*
+ ID_MODEL_FROM_DATABASE=ZBook 15
+
 pci:v00008086d00008C26sv000017AAsd0000220E*
  ID_MODEL_FROM_DATABASE=ThinkPad T440p
 
 pci:v00008086d00008C2D*
  ID_MODEL_FROM_DATABASE=8 Series/C220 Series Chipset Family USB EHCI #2
 
+pci:v00008086d00008C2Dsv0000103Csd00001909*
+ ID_MODEL_FROM_DATABASE=ZBook 15
+
 pci:v00008086d00008C2Dsv000017AAsd0000220E*
  ID_MODEL_FROM_DATABASE=ThinkPad T440p
 
 pci:v00008086d00008C31*
  ID_MODEL_FROM_DATABASE=8 Series/C220 Series Chipset Family USB xHCI
 
+pci:v00008086d00008C31sv0000103Csd00001909*
+ ID_MODEL_FROM_DATABASE=ZBook 15
+
 pci:v00008086d00008C31sv000017AAsd0000220E*
  ID_MODEL_FROM_DATABASE=ThinkPad T440p
 
@@ -70487,6 +70856,9 @@ pci:v00008086d00008C34*
 pci:v00008086d00008C3A*
  ID_MODEL_FROM_DATABASE=8 Series/C220 Series Chipset Family MEI Controller #1
 
+pci:v00008086d00008C3Asv0000103Csd00001909*
+ ID_MODEL_FROM_DATABASE=ZBook 15
+
 pci:v00008086d00008C3Asv000017AAsd0000220E*
  ID_MODEL_FROM_DATABASE=ThinkPad T440p
 
@@ -70547,6 +70919,9 @@ pci:v00008086d00008C4E*
 pci:v00008086d00008C4F*
  ID_MODEL_FROM_DATABASE=QM87 Express LPC Controller
 
+pci:v00008086d00008C4Fsv0000103Csd00001909*
+ ID_MODEL_FROM_DATABASE=ZBook 15
+
 pci:v00008086d00008C4Fsv000017AAsd0000220E*
  ID_MODEL_FROM_DATABASE=ThinkPad T440p
 
@@ -70820,6 +71195,9 @@ pci:v00008086d00009C02*
 pci:v00008086d00009C03*
  ID_MODEL_FROM_DATABASE=Lynx Point-LP SATA Controller 1 [AHCI mode]
 
+pci:v00008086d00009C03sv000017AAsd00002214*
+ ID_MODEL_FROM_DATABASE=ThinkPad X240
+
 pci:v00008086d00009C04*
  ID_MODEL_FROM_DATABASE=Lynx Point-LP SATA Controller 1 [RAID mode]
 
@@ -70907,12 +71285,18 @@ pci:v00008086d00009C1F*
 pci:v00008086d00009C20*
  ID_MODEL_FROM_DATABASE=Lynx Point-LP HD Audio Controller
 
+pci:v00008086d00009C20sv000017AAsd00002214*
+ ID_MODEL_FROM_DATABASE=ThinkPad X240
+
 pci:v00008086d00009C21*
  ID_MODEL_FROM_DATABASE=Lynx Point-LP HD Audio Controller
 
 pci:v00008086d00009C22*
  ID_MODEL_FROM_DATABASE=Lynx Point-LP SMBus Controller
 
+pci:v00008086d00009C22sv000017AAsd00002214*
+ ID_MODEL_FROM_DATABASE=ThinkPad X240
+
 pci:v00008086d00009C23*
  ID_MODEL_FROM_DATABASE=Lynx Point-LP CHAP Counters
 
@@ -70922,12 +71306,18 @@ pci:v00008086d00009C24*
 pci:v00008086d00009C26*
  ID_MODEL_FROM_DATABASE=Lynx Point-LP USB EHCI #1
 
+pci:v00008086d00009C26sv000017AAsd00002214*
+ ID_MODEL_FROM_DATABASE=ThinkPad X240
+
 pci:v00008086d00009C2D*
  ID_MODEL_FROM_DATABASE=Lynx Point-LP USB EHCI #2
 
 pci:v00008086d00009C31*
  ID_MODEL_FROM_DATABASE=Lynx Point-LP USB xHCI HC
 
+pci:v00008086d00009C31sv000017AAsd00002214*
+ ID_MODEL_FROM_DATABASE=ThinkPad X240
+
 pci:v00008086d00009C35*
  ID_MODEL_FROM_DATABASE=Lynx Point-LP SDIO Controller
 
@@ -70937,6 +71327,9 @@ pci:v00008086d00009C36*
 pci:v00008086d00009C3A*
  ID_MODEL_FROM_DATABASE=Lynx Point-LP HECI #0
 
+pci:v00008086d00009C3Asv000017AAsd00002214*
+ ID_MODEL_FROM_DATABASE=ThinkPad X240
+
 pci:v00008086d00009C3B*
  ID_MODEL_FROM_DATABASE=Lynx Point-LP HECI #1
 
@@ -70958,6 +71351,9 @@ pci:v00008086d00009C42*
 pci:v00008086d00009C43*
  ID_MODEL_FROM_DATABASE=Lynx Point-LP LPC Controller
 
+pci:v00008086d00009C43sv000017AAsd00002214*
+ ID_MODEL_FROM_DATABASE=ThinkPad X240
+
 pci:v00008086d00009C44*
  ID_MODEL_FROM_DATABASE=Lynx Point-LP LPC Controller
 
@@ -72557,6 +72953,54 @@ pci:v0000A727d00000013*
 pci:v0000A727d00006803*
  ID_MODEL_FROM_DATABASE=3CRDAG675B Wireless 11a/b/g Adapter
 
+pci:v0000AA00*
+ ID_VENDOR_FROM_DATABASE=iTuner
+
+pci:v0000AA01*
+ ID_VENDOR_FROM_DATABASE=iTuner
+
+pci:v0000AA02*
+ ID_VENDOR_FROM_DATABASE=iTuner
+
+pci:v0000AA03*
+ ID_VENDOR_FROM_DATABASE=iTuner
+
+pci:v0000AA04*
+ ID_VENDOR_FROM_DATABASE=iTuner
+
+pci:v0000AA05*
+ ID_VENDOR_FROM_DATABASE=iTuner
+
+pci:v0000AA06*
+ ID_VENDOR_FROM_DATABASE=iTuner
+
+pci:v0000AA07*
+ ID_VENDOR_FROM_DATABASE=iTuner
+
+pci:v0000AA08*
+ ID_VENDOR_FROM_DATABASE=iTuner
+
+pci:v0000AA09*
+ ID_VENDOR_FROM_DATABASE=iTuner
+
+pci:v0000AA0A*
+ ID_VENDOR_FROM_DATABASE=iTuner
+
+pci:v0000AA0B*
+ ID_VENDOR_FROM_DATABASE=iTuner
+
+pci:v0000AA0C*
+ ID_VENDOR_FROM_DATABASE=iTuner
+
+pci:v0000AA0D*
+ ID_VENDOR_FROM_DATABASE=iTuner
+
+pci:v0000AA0E*
+ ID_VENDOR_FROM_DATABASE=iTuner
+
+pci:v0000AA0F*
+ ID_VENDOR_FROM_DATABASE=iTuner
+
 pci:v0000AA42*
  ID_VENDOR_FROM_DATABASE=Scitex Digital Video
 
